assets/javascripts/discourse/helpers/format.js

```diff
@@ -0,0 +1,8 @@
+import Helper from "@ember/component/helper";
+import I18n from "I18n";
+
+export function format(number) {
```

Comment:
I feel like it's a little difficult to figure out the intent of this helper with the `format` name. What is the difference between using `{{ number '224.5' }}` and `{{ format '224.5' }}`? I think a more descriptive name for this helper might help, but if I try both of these locally I get the same result either way (225, it rounds up). Should there be more precision for this helper to show decimals?

Reply:
Also if we are adding a helper that is reusable, it should be added in core, not a plugin.

Reply:
The behaviour is what we want, we don't want to show decimals at all. Afaik, there won't even be decimals with cheers, right? It's always a whole point/cheer. The difference will only show from 1000 upwards:
```
      {{number 12345}} //used on mobile
      {{format 12345}} //used on desktop
```
will give:
<img width="165" alt="image" src="https://user-images.githubusercontent.com/101828855/165485142-8ac3cc6b-450e-4d9d-80cb-1e7c946923af.png">

I agree on the name, would you happen to have a suggestion?

Reply:
Can you think of a better name for the helper than `format`? Maybe `fullNumber`? 

I think for now we should keep the helper in the plugin, as you have it. I'd move it core only once we have at least one case of it being used in core.

Reply:
Nice thanks @Chapoi and @pmusaraj , `fullnumber` is better :+1: For future reference, the helpful details in the comment https://github.com/discourse/discourse-gamification/pull/19#discussion_r859570613 should be in the description of the PR, it helps a lot with context for reviewing :)